lib/diagnostic.rb

```diff
@@ -91,13 +91,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'pop(2)'
```

Comment:
arr.pop(2) or arr.pop 2

This would give an error without an object to use pop on.